python/test/ngraph/test_ops_binary.py

```diff
@@ -184,3 +184,22 @@ def test_binary_operators_with_scalar(operator, numpy_function):
     result = computation(value_a)
     expected = numpy_function(value_a, value_b)
     assert np.allclose(result, expected)
+
+
+def test_elu_operator():
+    runtime = get_runtime()
```

Comment:
Let's add one more test, where we pass in an `nd.array` and a scalar number as inputs instead of `ng.parameter`s